projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -69,9 +69,9 @@ The component has multiple responsibilities:
 - Using the service to perform a _side effect_, reaching out to an external API to fetch the movies
 - Changing the _state_ of the movies within the component.
 
-`Effects` when used along with `Store`, decrease the responsibility of the component.  In a larger application, this becomes more important as you have multiple sources of data, with multiple services required to fetch those pieces of data, and services potentially relying on other services.
+`Effects` when used along with `Store`, decrease the responsibility of the component.  In a larger application, this becomes more important. Because you have multiple sources of data, with multiple services required to fetch those pieces of data, and services potentially relying on other services.
```

Comment:
```suggestion
`Effects` when used along with `Store`, decrease the responsibility of the component.  In a larger application, this becomes more important because you have multiple sources of data, with multiple services required to fetch those pieces of data, and services potentially relying on other services.
```